scipy/stats/_continuous_distns.py

```diff
@@ -4089,6 +4089,63 @@ def _entropy(self):
 semicircular = semicircular_gen(a=-1.0, b=1.0, name="semicircular")
 
 
+class skew_normal_gen(rv_continuous):
+    """A skew-normal random variable
+    
+    %(before_notes)s
+    
+    Notes
+    -----
+    The pdf is
+
+    skewnormal.pdf(x, a) = 2*norm.pdf(x)*norm.cdf(ax)
+    
+    `skewnormal` takes ``a`` as a skewness parameter
+    When a=0 the distribution is identical to a normal.
```

Comment:
"... is identical to a normal." --- to a normal distribution? To a normal distribution, `scipy.stats.norm`?